allenact/base_abstractions/sensor.py

```diff
@@ -195,16 +195,16 @@ def _get_observation_space(self) -> gym.spaces.Box:
         return su.flatten_space(self.unflattened_observation_space)
 
     @lazy_property
-    def _zeroed_action(self):
-        return self.action_space.sample().zero_()
+    def _zeroed_observation(self):
+        return np.zeros_like(self.observation_space.sample())
 
     def get_observation(
         self, env: EnvType, task: SubTaskType, *args: Any, **kwargs: Any
     ) -> Any:
         # If the task is completed, we needn't (perhaps can't) find the expert
         # action from the (current) terminal state.
         if task.is_done():
-            return np.array([self._zeroed_action, False])
+            return np.array([self._zeroed_observation, False])
```

Comment:
I'm not sure this comment isn't duplicated, but I believe here we just need to do `return self._zeroed_observation`

Reply:
(because the observation space already includes the flag, right?)

Reply:
You're definitely correct!